test/internal/testlabels/labels.go

```diff
@@ -0,0 +1,32 @@
+package testlabels
+
+const (
+	// Kind is the label key used to store the primary kind that's being tested.
+	Kind         = "kind"
+	KindUDPRoute = "UDPRoute"
+)
+
+const (
+	// NetworkingFamily is the label key used to store the networking family of
+	// resources that are being tests.
+	//
+	// Possible, values: "gatewaypi", "ingress".
+	NetworkingFamily           = "networkingfamily"
+	NetworkingFamilyGatewayAPI = "gatewayapi"
+	NetworkingFamilyIngress    = "ingress"
+)
+
+const (
+	// GatewayAPISupportLevel is the label key used to store the support level
+	// of the Gateway API resources being tested.
+	GatewayAPISupportLevel      = "gatewayapisupportlevel"
+	GatewayAPISupportLevelAlpha = "alpha"
+	GatewayAPISupportLevelBeta  = "beta"
+)
```

Comment:
What's the purpose of these consts? As far as I can see they are unused

Reply:
Those labels are used in test's features for filtering/skipping e.g. https://github.com/Kong/kubernetes-ingress-controller/blob/1c2eb68f8c6b7f0879d1a80f262e21d049b185d3/test/integration/isolated/examples_udproute_test.go#L29-L31. 

This is also mentioned in the added TESTING.md section https://github.com/Kong/kubernetes-ingress-controller/blob/isolated-integration-tests/TESTING.md#how-to-run-3

---

I've removed - fb91c460f - the Gateway API support level labels because that could easily go outdated when e.g. we bump Gateway API version that changes the support level of say `UDPRoute` and we have it hardcoded in those labels (as far as I know there's no `const` or a helper function which we could use here to always refer to the same name so that the tests get updated values).

Reply:
👍